src/helpers/getReactNativeHostComponentNames.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Text, TextInput, View } from 'react-native';
+import { HostComponentNames } from '../config';
+import render from '../render';
+
+export function getReactNativeHostComponentNames(): HostComponentNames {
+  const { getByTestId } = render(
```

Comment:
BTW this will mess the current `screen` output, which might be problematic if user is using `screen` + `findBy*` queries which are essientially just `waitFor` + `getBy*`. In order to avoid it we need to expose some `internalRender` function that would skip setting `screen`. You may event want to render using `TestRenderer` and bind queries to the returned `root` using `getQueriesForElement`.

Reply:
Oh you're right I hadn't thought of this, good catch ! I like very much the idea of just using TestRenderer and binding queries, if there is further need for an internal render that would skip screen maybe it should get refactored but for now I think it would bring unnecessary complexity